src/code.c

```diff
@@ -53,9 +53,7 @@
 #include <src/hpc/aobjects.h>           /* atomic objects */
 
 #include <src/vars.h>                   /* variables */
-
-
-#include <src/profile.h>                /* access to stat register function */
+#include <src/hookintrprtr.h>           /* access to stat register function */
```

Comment:
I noticed that the comments after this include vary; sometimes it says "installing methods" instead of the above. Intentional?

(Note that on the long run, I think we should just get rid of these comments anyway, but as long as we have them...)